2697. Lexicographically Smallest Palindrome/solution.py

```diff
@@ -5,21 +5,38 @@ class Solution:
     # the alphabet then the respective letter in b does.
     def makeSmallestPalindrome(self, s: str) -> str:
         # These are the two indices that, in the resulting string, we are going to make the same.
-        # If the string length is odd, then the first index should start at the floored value
-        # and the second index should start at the ceiled value.
+        # If the string length is odd, then the center should be the middle letter, and the first
+        # and last indices surround it.
+        if (len(s) % 2 == 1):
```

Comment:
I think you probably could have simplified all of this logic with the integer divide operator (//)